src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/ToModelTransformers.kt

```diff
@@ -243,7 +246,7 @@ fun ProbationSearchResponseResult.transformToModelFoundProbationRecord(licence:
     name = "${name.forename} ${name.surname}".convertToTitleCase(),
     crn = licence?.crn,
     nomisId = licence?.nomsId,
-    comName = "${manager.name?.forename} ${manager.name?.surname}".convertToTitleCase(),
+    comName = if (manager.name?.forename.isNullOrBlank() || manager.name?.surname.isNullOrBlank()) null else "${manager.name?.forename} ${manager.name?.surname}".convertToTitleCase(),
```

Comment:
It looks like `manager.name` is the nullable thing?

If so, it's more idiomatic to:
`manage.name?.let { "${it.forename} ${it.surname}".convertToTitleCase()" }`

Reply:
Magic, thanks Andy.